test/lib.reviews.js

```diff
@@ -8,6 +8,7 @@ function assertValid (review) {
   assert.isString(review.id);
   assert(review.id);
   assert.isString(review.userName);
+  assert('userImage' in review);
   assert(review.userName);
```

Comment:
Ok...maybe for now I'll assert that it's there and if you notice any failures then we can be more defensive?

For one, I did see a review where the image url appeared to link to a generic profile picture, so I believe anonymous/no-pic reviews should still have a url.